docs/source/configuration/index.rst

```diff
@@ -1,4 +1,4 @@
-.. _atlas_config:
+.. _atlas-config:
```

Comment:
derp. When you build the docs, it only shows you the first occurrence of a broken ref, so I fixed one of them, only to find that there were 3 more!